pkg/app/server.go

```diff
@@ -157,23 +174,21 @@ func RunKubeStateMetrics(ctx context.Context, opts *options.Options, factories .
 	}
 
 	var resources []string
```

Comment:
since we know the size of the array, it would be better to initialize to the correct size directly, I would avoid having to resize it multiple times with `append`

Reply:
Good catch, fixed it!